src/components/modules/nutrition/add-food/food-list-item-info.vue

```diff
@@ -0,0 +1,99 @@
+<template>
```

Comment:
HMTL can be quite heavily nested sometimes so maybe you could consider using smaller intendation e.g. 2 spaces? Only in HTML files.

Reply:
I will check whether I can change indentation in a file for HTML only

Reply:
Done